source/appModules/foobar2000.py

```diff
@@ -41,11 +43,18 @@ def parseIntervalToTimestamp(interval):
 	return calendar.timegm(time.strptime(interval.strip(), format))
 
 class AppModule(appModuleHandler.AppModule):
-	statusBar=None
 
-	def event_gainFocus(self, obj, nextHandler):
-		if not self.statusBar: self.statusBar=api.getStatusBar()
-		nextHandler()
+	def _get_statusBar(self):
+		# #11082: retrieve status bar handle and resulting NVDA object from playlist window.

```

Comment:
Could you add a comment to this method to explain why this implementation is preferred over the standard statusbar lookup.